test/crypten/test_context.py

```diff
@@ -13,15 +15,24 @@ def test_context(workers):
     alice_tensor_ptr = th.tensor([42, 53]).tag("crypten_data").send(alice)
     bob_tensor_ptr = th.tensor([101, 32]).tag("crypten_data").send(bob)
 
-    @run_multiworkers([alice, bob], master_addr="127.0.0.1")
+    @sy.func2plan()
+    def toy_func():
+        # crypten.load is actually syft_load
+        alice_tensor = crypten.load("crypten_data", 1, "alice", size=(2, ))
+        bob_tensor = crypten.load("crypten_data", 2, "bob", size=(2, ))
+
+        result = alice_tensor + bob_tensor
+        result = result.get_plain_text()
```

Comment:
The load function shouldn't fail and it would return a real `cryptensor`, right? Then it's not the fake `get_plain_text` that's called here.

Reply:
Hmm, when the plan is building the  ```load``` should go on the Exception branch from here:
```
try:
            response = func(*args, **kwargs)
        except Exception as e:
            response = th.rand(kwargs["size"])
```
And it should be a dummy response (only to have the shape of the response) and the result should also be a ```        th.randn(self.shape)``` from ```syft/generic/tensor```